modules/brainbrowser/templates/form_brainbrowser.tpl

```diff
@@ -184,3 +184,38 @@
     document.getElementById('brainbrowserPage')
   ).render(RBrainBrowser());
 </script>
+
```

Comment:
I don't understand why all this is necessary. Can't you just do `{dgettext("brainbrowser", "Threshhold")}` in the smarty template?